### PR TITLE
[script.module.enum34] 1.1.6

### PR DESCRIPTION
**Pull request by jmooremcc**
### Description

An enumeration is a set of symbolic names (members) bound to unique, constant values
This module is the backported version of the enum34 package that runs under Python 2.7
Do not use this module with Python versions greater than 3.3

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.enum34/addon.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.enum34" version="1.1.6" name="enum" provider-name="ethanf">
```
Comment (Rechi) [inline script.module.enum34/addon.xml]: upstream author used `Ethan Furman` as author name

File: script.module.enum34/addon.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <requires>
+        <import addon="xbmc.python" version="2.26.0" />
+    </requires>
+    <extension library="lib/enum" point="xbmc.python.module" />
```
Comment (Rechi) [inline script.module.enum34/addon.xml]: I think it has to be `library="lib"` to work correct, e.g `from enum import Enum`

File: script.module.enum34/addon.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">A Python interface for enum34</summary>
```
Comment (Rechi) [inline script.module.enum34/addon.xml]: upstream uses `Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4` as short description (summary)

File: script.module.enum34/addon.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        <description lang="en_GB">A Python interface for enum34</description>
+        <website>https://pypi.org/project/enum34/</website>
+        <license>BSD</license>
+        <platform>all</platform>
+        <source>https://bitbucket.org/stoneleaf/enum34/src/default/</source>
```
Comment (Rechi) [inline script.module.enum34/addon.xml]: https://bitbucket.org/stoneleaf/enum34

File: script.module.enum34/addon.xml (hunk 1 above)
Comment (jmooremcc) [reply script.module.enum34/addon.xml]: Done

File: script.module.enum34/addon.xml (hunk 3 above)
Comment (jmooremcc) [reply script.module.enum34/addon.xml]: Done

File: script.module.enum34/addon.xml (hunk 4 above)
Comment (jmooremcc) [reply script.module.enum34/addon.xml]: Done

File: script.module.enum34/addon.xml (hunk 2 above)
Comment (jmooremcc) [reply script.module.enum34/addon.xml]: I tested it and you're correct.
I've made the change.

**Comment by enen92** [comment]
can you squase the commits into a single one? Thanks

**Comment by jmooremcc** [comment]
> can you squase the commits into a single one? Thanks

I hope I've done the squash correctly.
I've learned a lot about git over the past few days and want to thank you for patience and your help.

**Pull request closed without merging**